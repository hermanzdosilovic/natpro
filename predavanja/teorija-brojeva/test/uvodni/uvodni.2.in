11
100
200
2000
1337
13337
12345
123456
200000
1000000
999999
999000
