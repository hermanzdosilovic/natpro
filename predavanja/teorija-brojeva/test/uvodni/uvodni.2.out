4
2
8
8
8
4
2
4
4
4
8
